src/components/MessagesList/MessagesList.vue

```diff
@@ -302,12 +289,13 @@ export default {
 		 * a long-polling request for new messages.
 		 */
 		startGettingMessages() {
-			this.isInitiated = false
 			this.getMessages()
```

Comment:
Now that `startGettingMessages` just calls `getMessages` you could replace calls to `startGettingMessages` with calls to `getMessages` and remove `startGettingMessages` :-) (but move its documentation to `getMessages` ;-) ).

Reply:
sure